tokio/src/macros/pin.rs

```diff
@@ -140,5 +140,5 @@ macro_rules! pin {
             let $x = $init;
             crate::pin!($x);
         )*
-    };
+    }
```

Comment:
nit: this looks like an unrelated change?

Reply:
failed merge!